CONTRIBUTING.md

```diff
@@ -191,6 +191,11 @@ supported:
 * `upload` (optional): This field is set to `false` to prevent automatic
   uploads to PyPI. This should only used in special cases, e.g. when the stubs
   break the upload.
+* `partial_stub` (optional): This field is used to let stub_uploader know whether
+  to distribute a type stub package as
+  [partial](https://peps.python.org/pep-0561/#partial-stub-packages). This is for
+  3rd-party stubs that don't cover the entirety of the package's public API.
+  In most cases, this field follows `stubtest_requirements`.
```

Comment:
Did you mean `ignore_missing_stub`? I would also not mention the stub uploader as that is mostly irrelevant for usual contributors. Also maybe word it as "... is identical to ..." Otherwise it sounds a bit as if this field must be placed after the other field physically.

```suggestion
* `partial_stub` (optional): This field marks the type stub package as
  [partial](https://peps.python.org/pep-0561/#partial-stub-packages). This is for
  3rd-party stubs that don't cover the entirety of the package's public API.
  In most cases, this field is identical to `ignore_missing_stub`.
```

(Sidenote: I just noticed that we document neither `ignore_missing_stub` nor `stubtest_requirements` in `CONTRIBUTING`.)

Reply:
oops thanks.